clang/test/Analysis/security-deprecated-buffer-handling-allow-without-c11.c

```diff
@@ -0,0 +1,35 @@
+// Test 1: Without C11 and without flag - should NOT warn
+// RUN: %clang_analyze_cc1 %s -verify=c99-noflag -std=gnu99 \
```

Comment:
So we use `gnu99`; couldn't we use the standard `c99` language flavour instead?
I think the gnu variant allows the gnu extensions, which leads to a strictly broader l language flavour than standard C. Is this intentional?